docs/useGetRecordRepresentation.md

```diff
@@ -53,6 +53,17 @@ const PostEdit = () => (
 )
 ```
 
+## Preferences
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why "Preferences"?